src/Components/ContextMenu.re

```diff
@@ -142,14 +141,21 @@ module Menu = {
       backgroundColor(theme.menuBackground),
       color(theme.menuForeground),
       width(Constants.menuWidth),
+      boxShadow(
+        ~xOffset=-5.,
+        ~yOffset=-5.,
+        ~blurRadius=25.,
+        ~spreadRadius=-10.,
+        ~color=Color.rgba(0., 0., 0., 0.0001),
+      ),
```

Comment:
It works! But it's also kind of broken. The alpha value of the color doesn't seem to be respected, and there seems to be some built-in offset that has to be countered by specifying negative offsets.

I managed to tone it down a bit by adding a negative spread, but it still looks a bit off.

Reply:
> It works!

Hooray! 🎉 

> But it's also kind of broken. The alpha value of the color doesn't seem to be respected, and there seems to be some built-in offset that has to be countered by specifying negative offsets.

Ya, unfortunately there are bugs and quirks with the [current implementation](https://github.com/revery-ui/revery/blob/50e50c3a3dd56a2e835f427c530940b7e7b601de/src/Draw/GradientShader.re#L55). 

Skia will give us a more robust set of primitives for stuff like this!

Reply:
Thanks for the pointer! Looks like fixing it so the alpha value is taken into account shouldn't be too hard, just pass in a vec4 instead of a vec3 and multiple the alpha value with the blur value. But the offset bug seems trickier. I can't se where that comes from. That's also less of an issue I think though, so it can definitely wait for Skia.